src/test/java/org/folio/rest/api/HridSettingsIncreaseMaxValueMigrationTest.java

```diff
@@ -0,0 +1,120 @@
+package org.folio.rest.api;
+
+import static org.folio.rest.api.StorageTestSuite.TENANT_ID;
+import static org.folio.rest.api.StorageTestSuite.getVertx;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import org.folio.rest.persist.PostgresClient;
+import org.folio.util.ResourceUtil;
```

Comment:
We should create a direct `test` dependency to `util` in the pom file.

Reply:
I've added the dependency, but the scope have to be `compile` because the utilities is used by maven plugins and RMB itself.

Reply:
Yeah, the dependencies in this project, as well as most FOLIO projects, are not managed well. The `GenerateRunner` main class we call should really be a fat jar so we could include a direct runtime dependency on `domain-models-interface-extensions`. Since it is not, as you've mentioned, we can't limit the scope of `utils` to test. Interestingly, if we do add a runtime dependency on `domain-models-interface-extensions`, `ShelfLocation` breaks. It is using a dependency on the legacy `commons-lang` that is available via `domain-models-interface-extensions`, but no where else, since we want to use `commons-lang3`. There are probably other problems with the dependencies that a maven plugin would expose (just tried and there are many missing dependencies and one unused one). I guess we should cross that tech-debt bridge at a later date.

Reply:
Yeah, alas it really is. I didn't know there was a maven plugin for detecting dependency issues until a few years into the project